index.html

```diff
@@ -341,19 +341,24 @@ <h2><code>Window</code> interface extensions</h2>
 </pre>
     <p>Each <a>Window</a> has:</p>
     <ul>
-      <li>A <dfn>list of idle request callbacks</dfn>. The list MUST be
-      initially empty and each entry in this list is identified by a number,
-      which MUST be unique within the list for the lifetime of the <code>Window</code>
-      object.</li>
-      <li>A <dfn>list of runnable idle callbacks</dfn>. The list MUST be
-      initially empty and each entry in this list is identified by a number,
-      which MUST be unique within the list of the lifetime of the <code>Window</code>
-      object.</li>
       <li>An <dfn>idle callback identifier</dfn>, which is a number which MUST
       initially be zero.</li>
+    </ul>
+
+    <p>In addition, each <a>event loop</a> has:</p>
```

Comment:
I think it would be cleaner to use the similar-origin window agent here as the ordering is only supposed to be observable within such an agent. That's also what we do for mutation observers.

Reply:
Agreed, that makes sense. Is there an easy way to get the all the similar-origin window agents associated with an event loop (i.e., for step 11 of the event loop processing model)?

Reply:
No concrete `<dfn>` you can link to. But IMO something like

> all similar-origin window agents that use _event loop_

is well-defined, given the definition at https://html.spec.whatwg.org/#window-event-loop

Reply:
Great, I've switch this over to similar-origin window agent.